layouts/docs/docsportal_home.html

```diff
@@ -2,144 +2,48 @@
     {{ if not .Params.notitle }}
 	 <h1>{{ .Title }}</h1>
     {{ end }}
-    {{ if eq .Lang "en" }}
     {{ template "docs-portal-cards" . }}
-    {{ end }}
 {{ end }}
 
 {{ define "content-id" }}content{{ end }}
 
 {{ define "docs-portal-cards" }}
```

Comment:
@gochist, Looks good.
Do you know if the `content-id` define is required? 
Suggestion: I initially was confused by the similar template names, `docs-portal-cards` and `docs-portal-card`.
Could rename `docs-portal-cards` to something like, `docs-portal-content` and rename the template that renders all the cards to `docs-portal-cards`.

Reply:
@kbhawkey Thank you! I addressed your review. I'm not sure what the `content-id` is for. But when I delete it, the page layout is broken.

Reply:
@kbhawkey Great feedback--I had the same confusion. 👍